### DISCUSSION TITLE
Proxy only one URL

### DISCUSSION THREAD

**Discussion by vmanju123** [mitmproxy/mitmproxy#5862] [Q&A]
Hi Everyone

here is my use case
I am running a MITM proxy on a windows 10 system
I have set the system wide proxy settings to http://localhost:8080, so all https traffic from all apps and browsers is routed via the local proxy on same system

I would like MITM proxy to only intercept https traffic on one URL
i don't intend to modify anything in the https request or the response, I just want to monitor only that URL
I want all other traffic (other than the URL specified ) to be bypassed by the proxy meaning skip the proxy and ignore everything

is that possible ?

I have used mitmproxy --intercept URL command, but it appears the intercept command is to intercept something with intention to modify something, but i dont intend to modify any communication but just monitor both request and response

I also want to run this in command line not in UI with interaction
guidance is appreciated

thanks
MV

thanks
Manjunath

**Comment by mhils** [comment]
You can only intercept specific hosts by using the allow_hosts/ignore_hosts options. The decision to intercept is made based on the hostname only (the full URL is only accessible after decryption). But you can certainly filter the view to only show one particular request.

**Comment by vmanju123** [comment]
thanks that worked as expected, is there a way to allow only a URL than a host ?
